src/test/java/orangetest/sqltask/SqlDataProvider.java

```diff
@@ -0,0 +1,136 @@
+package orangetest.sqltask;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SqlDataProvider extends DataProvider {
```

Comment:
one SqlDataProvider for Emps and Vacancies breaks SRP - pliz split it into 2 providers

Reply:
fixed